package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-ui-kit",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-ui-kit/issues",
-  "version": "4.4.1",
+  "version": "4.5.0",
```

Comment:
Decided to do 4.5.0 instead of 4.4.1 because those color changes could interfere with things like the corona banner. (button may not appear clickable anymore? something like that)

Reply:
probably will, let's roll this update on core in a separate PR as discussed to see the scope of brokenness